spec/fixtures/sources/commonjs-grouped/index.js

```diff
@@ -0,0 +1,7 @@
+const functions = require("../../../../src/index");
+
+exports.v1http = functions.https.onRequest((_req, resp) => {
```

Comment:
While I get that you're prefixing params with _ to indicate they're unused, that seems to violate style guides for naming.

Reply:
Fixed!